safehttp/plugins/csp/csp_test.go

```diff
@@ -126,11 +126,13 @@ func TestBefore(t *testing.T) {
 		wantEnforcePolicy    []string
 		wantReportOnlyPolicy []string
 		wantNonce            string
+		wantXFO              []string
```

Comment:
Can you please add a test to check that we don't set XFO if no framing policy is specified and if the framing policy is RO?